src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -10650,7 +10669,13 @@ namespace HVACUnitarySystem {
 			} // ELSE ! No Moisture Load
 		} // No Heating/Cooling Load
 
-		MSHPMassFlowRateLow = CompOffMassFlow; // these need to be set for multi-speed coils
+		if ( UnitarySystem( UnitarySysNum ).MultiSpeedHeatingCoil && ( HeatingLoad && HeatSpeedNum == 1 ) ) {
+			MSHPMassFlowRateLow = CompOnMassFlow * PartLoadRatio; // proportional to PLR when speed = 1,  #5518
+		} else if ( UnitarySystem( UnitarySysNum ).MultiSpeedCoolingCoil && ( CoolingLoad && CoolSpeedNum == 1 ) ) {
+			MSHPMassFlowRateLow = CompOnMassFlow * PartLoadRatio; // proportional to PLR when speed = 1,  #5518
+		} else {
+			MSHPMassFlowRateLow = CompOffMassFlow; // these need to be set for multi-speed coils
+		}
```

Comment:
Actually, var spd coils doesn't use this global (MSHPMassFlowRate*) so this shouldn't hurt.